Package/Installer.cs

```diff
@@ -215,7 +239,11 @@ internal int RunCommand(string command, bool force, bool modifiesPackageFiles)
                     return (int)ExitCodes.UserCancelled;
 
                 log.Debug(ex);
-                return (int) ExitCodes.GeneralException;
+                return (int)ExitCodes.GeneralException;
+            }
+            finally
+            {
+                postInstallHook();
```

Comment:
It seems that postInstallHook is never actually used for anything, so maybe just delete it.